accname/name/comp_label.html

```diff
@@ -11,11 +11,140 @@
+<h6 aria-label="label" data-expectedlabel="label" data-testname="label valid on h6 element" class="ex">x</h6>
+<img alt="" aria-label="label" data-expectedlabel="label" data-testname="label valid on img element" class="ex" />
+<a href="" aria-label="label" data-expectedlabel="label" data-testname="label valid on link element" class="ex">x</a>
+<ul aria-label="label" data-expectedlabel="label" data-testname="label valid on list (unordered) element" class="ex">x</ul>
+<ol aria-label="label" data-expectedlabel="label" data-testname="label valid on list (ordered) element" class="ex">x</ol>
+<li aria-label="label" data-expectedlabel="label" data-testname="label valid on listitem element" class="ex">x</li>
+<main aria-label="label" data-expectedlabel="label" data-testname="label valid on main element" class="ex">x</main>
+<math aria-label="label" data-expectedlabel="label" data-testname="label valid on math element" class="ex">x</math>
+<meter aria-label="label" data-expectedlabel="label" data-testname="label valid on meter element" class="ex">x</meter>
+<nav aria-label="label" data-expectedlabel="label" data-testname="label valid on nav element" class="ex">x</nav>
+
+<!-- <option aria-label="label" data-expectedlabel="label" data-testname="label valid on option element" class="ex">x</option> -->
+<input type="radio" aria-label="label" data-expectedlabel="label" data-testname="label valid on input type radio element" class="ex" />
+<section aria-label="label" data-expectedlabel="label" data-testname="label valid on section element" class="ex">x</section>
+<tr aria-label="label" data-expectedlabel="label" data-testname="label valid on tr element" class="ex">x</tr>
+<tbody aria-label="label" data-expectedlabel="label" data-testname="label valid on tbody element" class="ex">x</tbody>
+<tfoot aria-label="label" data-expectedlabel="label" data-testname="label valid on tfoot element" class="ex">x</tfoot>
+<thead aria-label="label" data-expectedlabel="label" data-testname="label valid on thead element" class="ex">x</thead>
+<th scope="row" aria-label="label" data-expectedlabel="label" data-testname="label valid on th (scope row) element" class="ex">x</th>
+<input type="search" aria-label="label" data-expectedlabel="label" data-testname="label valid on input type search element" class="ex" />
+<hr aria-label="label" data-expectedlabel="label" data-testname="label valid on hr element" class="ex" />
+<input type="text" aria-label="label" data-expectedlabel="label" data-testname="label valid on input type text element" class="ex" />
+<textarea aria-label="label" data-expectedlabel="label" data-testname="label valid on textarea element" class="ex">x</textarea>
+
+
+<!-- TODO write tests that purposefully fail?? -->
+<!-- 1. if the current node has an aria-label attribute whose value is not undefined, return the value of aria-label -->
+<!-- 2. if the current node has an aria-label attribute whose value is not the empty string, return the value of aria-label -->
```

Comment:
My thought was, should I make a test where it _is_ an empty string to make sure it does fail?

Reply:
Ah yeah. I think one of the `<img>` tests I included covered the empty string, but perhaps not the all-whitespace string that will be trimmed to become the empty string. Good idea to add another.

Reply:
Of note, `&nbsp;` is not included in ASCII Whitespace so that would likely be preserved in an `aria-label` test.

See https://github.com/web-platform-tests/wpt/pull/42407/files#diff-6870d82f11ff11cf7c7b544756cecfdac2046acbfe2fbb0640e6d415fbf99916

Reply:
Included as part of e46c133360b2f7137c718248ee12f5b2a9216c4a